streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterTimelineProvider.java

```diff
@@ -176,7 +173,7 @@ public StreamsResultSet readCurrent() {
     }
 
     protected Queue<StreamsDatum> constructQueue() {
-        return Queues.synchronizedQueue(new LinkedBlockingQueue<StreamsDatum>(MAX_NUMBER_WAITING));
+        return new LinkedBlockingQueue<StreamsDatum>(MAX_NUMBER_WAITING);
```

Comment:
We hold a lock around any operations on this queue.  It doesn't need to be synchronized.